src/test/java/uk/gov/hmcts/ccd/domain/service/aggregated/MergeDataToSearchResultOperationTest.java

```diff
@@ -11,6 +11,7 @@
 import org.junit.jupiter.api.DisplayName;
```

Comment:
we forgot to add a good bunch of integration test which for a complex feature like this are fundamental